templates/external_cloud_setup.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+if [ "$#" -ne 1 ]; then
+    echo "One param for cluster-name needed"
+    echo "The usage: external_cloud_setup.sh <cluster-name>"
+    exit -1
```

Comment:
`-1` is out of range according to https://tldp.org/LDP/abs/html/exitcodes.html#EXITCODESREF
```suggestion
    exit 1
```